test/handler/test_fortitude_handler.vader

```diff
@@ -0,0 +1,74 @@
+Before
```

Comment:
```suggestion
Before:
```

The colon was missing here so this broke. I should be able to fix this by applying the suggestion.